src/util.ts

```diff
@@ -1,10 +1,16 @@
-import { AxiosError } from 'axios';
+import axios, { AxiosError } from 'axios';
 
 export function serializeObject(obj: unknown): object {
   if (obj && typeof obj === 'object') {
     return Object.getOwnPropertyNames(obj).reduce((map, key) => {
-      // eslint-disable-next-line no-param-reassign
-      map[key] = obj[key];
+      const propertyValue = obj[key];
+      if (key === 'error' && axios.isAxiosError(propertyValue)) {
+        // eslint-disable-next-line no-param-reassign
+        map[key] = propertyValue?.response?.data;
```

Comment:
I'm not sure I understand how this works. Why the `key === 'error` and why that key takes the response body from axios error?

Reply:
no no .. it was supposed to be `obj[key]` 😅